packages/oui-checkbox/src/checkbox.controller.js

```diff
@@ -9,6 +11,8 @@ export default class {
     }
 
     $postLink () {
+        addBooleanParameter(this, "required");
```

Comment:
Like the disabled attribute, it should be in `$onInit`. And if we use this helper method, we can change the check on disabled with `addBooleanParameter(this, "disabled")` too :)

Reply:
Oops yes, it must be in `$onInit`.